client/src/AuthRoutes/components/Formik/styles/Input.style.js

```diff
@@ -0,0 +1,17 @@
+import {makeStyles} from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+	formGroup: {
+		marginBottom: 20,
+	},
+	label: {
+		fontWeight: 900,
+		fontSize: 12,
+	},
+	input: {
+		margin: '3px 0',
+		borderRadius: '0',
+	},
+}));
```

Comment:
You could move these into the component file or leave them separate but you should consult with the team about preferred convention

Reply:
I agree with Dan, I think it would be better to have the useStyles in the same file as the component. That's how I usually set it up. I find that it makes it easier to make small adjustments without having to switch back and forth between files.

Reply:
Thanks Dan. 
I will add styles to components and get rid of the extra files and folders.
About prettier, we discussed on standup meetings, one of teammates is going to make a pull request with prettier configuration, and I guess the problem will be solved after his merge.
Other than styles being separate , the other structures like having pages and components folders inside each section, I discussed  it with Yassine, I guess this works for him too, I will check again with the team to make sure everyone is comfortable with the structure os src folder.
 I will make a commit with these changes this afternoon after our standup meeting.